beacon_node/network/src/sync/simple_sync.rs

```diff
@@ -407,6 +421,13 @@ impl<T: BeaconChainTypes> SimpleSync<T> {
             })
             .collect();
 
+        // ssz-encode the headers
+        //TODO: Make this more elegant
```

Comment:
This is currently what the encoding looks like.. and the encoding is done within SimpleSync, to unify the call to `send_rpc`. If anyone can think of a better way, let me know